src/OpenTelemetry.Instrumentation.AspNet/AspNetInstrumentationOptions.cs

```diff
@@ -43,5 +43,13 @@ public class AspNetInstrumentationOptions
         /// The type of this object depends on the event, which is given by the above parameter.</para>
         /// </remarks>
         public Action<Activity, string, object> Enrich { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to record http server attributes or not.
```

Comment:
I think it might be helpful if we add `Default value: False.` on the end of the summary.

Reply:
Thanks, added